web-app/src/components/Templates/AuthPages.vue

```diff
@@ -1,12 +1,37 @@
 <script setup lang="ts">
+  /* Components */
+  import type { PropType } from 'vue';
+  import Title from '../atoms/Title.vue';
+  import ButtonWithLoading from '../molecules/ButtonWithLoading.vue';
   import AuthPagesHeader from '../organisms/AuthPagesHeader.vue';
-  defineProps(['text', 'linkText', 'linkUrl']);
+
+  /* Props */
+  defineProps({
+    formTitle: { type: String, required: true },
+    formButtonText: { type: String, required: true },
+    onFormSubmit: { type: Function as PropType<(event: MouseEvent) => void>, required: true },
+    headerText: { type: String, required: true },
+    headerLinkText: { type: String, required: true },
+    headerLinkUrl: { type: String, required: true },
+    isLoading: { type: Boolean, default: false }
+  });
 </script>
 
 <template>
   <main class="AuthPagesWrapper">
-    <AuthPagesHeader :text="text" :linkText="linkText" :linkUrl="linkUrl" />
-    <slot></slot>
+    <AuthPagesHeader :text="headerText" :linkText="headerLinkText" :linkUrl="headerLinkUrl" />
+    <form>
+      <Title :text="formTitle" />
+      <div class="form-fields">
+        <slot></slot>
+      </div>
+      <ButtonWithLoading
+        :is-loading="isLoading"
+        :text="formButtonText"
+        button-type="submit"
+        :onClick="onFormSubmit"
```

Comment:
Props passed using kebab-case as recommended in the documentation. 
The same pattern is not followed above - I'll look for a lint issue to enforce this at a later point